jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -2516,7 +2627,7 @@
 						}
 					],
 					"request": {
-						"url": "{{url}}/v1.0/media;name=jdk1.8.0_92",
+						"url": "{{url}}/v1.0/media;name=Postman-Test-Tomcat-Media",
```

Comment:
Does changing this from a jdk to tomcat media have any affects on other tests?

Reply:
jdk1.8.0_92 is media created by ctp health check cookbook. So, removed the dependency and used postman to create one